### ISSUE TITLE
Startup Error: 

### ISSUE THREAD

**Issue by Pawtiko** [enigma-dev/lgmplugin#35]
Operating System: Windows 8.1
Version: 6.3
Architecture: amd64

Java Vendor: Oracle Corporation
Version: 1.8.0_45

Available processors (cores): 4
Free memory (bytes): 201705520
Maximum memory (bytes): 932184064
Total memory available to JVM (bytes): 257425408

File system root: C:\
Total space (bytes): 499999993856
Free space (bytes): 237344251904
Usable space (bytes): 237344251904

File system root: D:\
Total space (bytes): 0
Free space (bytes): 0
Usable space (bytes): 0

Stack trace:

java.lang.UnsatisfiedLinkError: Unable to load library 'compileEGMf': Native library (win32-x86-64/compileEGMf.dll) not found in resource path ([file:/C:/Users/Patrick/Desktop/ENIGMA/enigma-dev/plugins/enigma.jar])
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:271)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:398)
	at com.sun.jna.Library$Handler.<init>(Library.java:147)
	at com.sun.jna.Native.loadLibrary(Native.java:412)
	at com.sun.jna.Native.loadLibrary(Native.java:391)
	at org.enigma.EnigmaRunner.attemptLib(EnigmaRunner.java:247)
	at org.enigma.EnigmaRunner.access$1(EnigmaRunner.java:240)
	at org.enigma.EnigmaRunner$3.run(EnigmaRunner.java:178)

**Comment by RobertBColton** [comment]
@Pawtiko Can you please tell me the output of java -version from command prompt? Are you running a 64 bit Java installation? The ENIGMA plugin using JNA currently only supports 32 bit on Windows.

**Comment by Pawtiko** [comment]
it was 64 bit, so that must be the problem

**Comment by RobertBColton** [comment]
@Pawtiko Please try installing 32 bit Java in parallel and running it and let me know if that corrects it.

**Comment by RobertBColton** [comment]
Closing as duplicate of #20
